ingestion/src/metadata/ingestion/source/dashboard/metabase/metadata.py

```diff
@@ -40,6 +40,8 @@
 from metadata.utils.helpers import get_standard_chart_type, replace_special_with
 from metadata.utils.logger import ingestion_logger
 
+from .client import MetabaseClient
+
 HEADERS = {"Content-Type": "application/json", "Accept": "*/*"}
```

Comment:
`HEADERS` will not be used in this class as we have moved all api calls to client so let's clean it